client/my-sites/upgrades/components/domain-warnings/pending-gapps-tos-notice.jsx

```diff
@@ -10,6 +11,7 @@ import analyticsMixin from 'lib/mixins/analytics';
 import Notice from 'components/notice';
 import NoticeAction from 'components/notice/notice-action';
 import support from 'lib/url/support';
+import paths from 'my-sites/upgrades/paths';
```

Comment:
Maybe we could just import `domainManagementEmail` since that seems to be all we need here.

`import { domainManagementEmail } from 'my-sites/upgrades/paths';`